src/raftstore/store/peer.rs

```diff
@@ -172,6 +173,74 @@ impl<'a, T> ReadyContext<'a, T> {
     }
 }
 
+#[derive(Default)]
+struct SplitStates {
```

Comment:
This is too complicated. Why not just store the last committed split index?

Reply:
Because there may be many on-going split commands and we do not know which one will be committd and which on will be successfully executed.